target/linux/realtek/files-6.12/drivers/net/dsa/rtl83xx/dsa.c

```diff
@@ -458,14 +458,18 @@ static int rtl93xx_pcs_validate(struct phylink_pcs *pcs,
 		phylink_set(mask, 10000baseKR_Full);
 		phylink_set(mask, 10000baseSR_Full);
 		phylink_set(mask, 10000baseCR_Full);
+		phylink_set(mask, 10000baseLR_Full);
 	}
-	if (state->interface == PHY_INTERFACE_MODE_INTERNAL) {
+	if (state->interface == PHY_INTERFACE_MODE_INTERNAL ||
+	    state->interface == PHY_INTERFACE_MODE_10GBASER ||
+	    state->interface == PHY_INTERFACE_MODE_1000BASEX) {
 		phylink_set(mask, 1000baseX_Full);
 		phylink_set(mask, 1000baseT_Full);
 		phylink_set(mask, 10000baseKR_Full);
 		phylink_set(mask, 10000baseT_Full);
 		phylink_set(mask, 10000baseSR_Full);
 		phylink_set(mask, 10000baseCR_Full);
+		phylink_set(mask, 10000baseLR_Full);
```

Comment:
This function was dropped with https://git.openwrt.org/?p=openwrt/openwrt.git;a=commit;h=9533e2e574c73320f0410c763b2ec658b8b91639

This commit is obsolete.